### PR TITLE
feat: Adding output for private natgateway

### PR DESCRIPTION
**Pull request by tommoatki1994**
## Description
feat: Adding output using private natgateway

## Motivation and Context
fixes minor issue

## How Has This Been Tested?
- [x] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

**Review by hammy414 — COMMENTED** [review]
This would be very helpful to come up to date with the pub VPC features.

**Comment by Smana** [comment]
tf-controller plan output:

```hcl

No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration
and found no differences, so no changes are needed.

```

To apply this plan, please **merge** this pull request.

**Comment by Smana** [comment]
tf-controller plan output:

```hcl

No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration
and found no differences, so no changes are needed.

```

To apply this plan, please **merge** this pull request.

File: outputs.tf [hunk 1]
```diff
@@ -129,7 +129,10 @@ output "public_route_table_ids" {
   description = "List of IDs of public route tables"
   value       = local.public_route_table_ids
 }
-
+output "private_nat_gateway_ips" {
+  description = "List of private nat gateway ip's"
+  value       = aws_nat_gateway.this[*].private_nat_gateway_private_ips
```
Comment (bryantbiggs) [inline outputs.tf:134]: this does not appear to be a valid attribute - where are you seeing this?

**Pull request closed without merging**